node/state.rei

```diff
@@ -12,6 +12,7 @@ module Address_map: Map.S with type key = Address.t;
 module Uri_map: Map.S with type key = Uri.t;
 type t = {
   identity,
+  trusted_validators: Validators.t,
```

Comment:
This should definitely not be a `Validators.t`

Reply:
Addressed in the incoming commit